src/LSST/system/ComConnection.cpp

```diff
@@ -162,6 +154,7 @@ void ComConnection::_sendWelcomeMsg() {
     {
         json js;
         js["id"] = "summaryState";
+        globals.setSummaryState(5);
```

Comment:
What is `5` here? If it could be `enum` of State, why not use the `enum` to have the higher readability?